data/src/emulator.js

```diff
@@ -240,6 +240,16 @@ class EmulatorJS {
         this.canvas.classList.add('ejs_canvas');
         this.videoRotation = ([0, 1, 2, 3].includes(this.config.videoRotation)) ? this.config.videoRotation : this.preGetSetting("videoRotation") || 0;
         this.videoRotationChanged = false;
+        this.config.screenshot = this.config.screenshot || {};
+        this.config.screenshot.source = (this.config.screenshot.source === "canvas" || this.config.screenshot.source === "retroarch") ? this.config.screenshot.source : "canvas";
```

Comment:
`["canvas", "retroarch"].includes(this.config.screenshot.source)`

much smaller